jobserver/migrations/0008_coredeveloper_to_staffareaadministrator.py

```diff
@@ -0,0 +1,44 @@
+# Generated by Django 5.1.1 on 2024-10-02 11:00
```

Comment:
I think this is mostly hand-written and should have a docstring explaining why it exists, what the purpose and rationale are, and the approach taken. If anyone need to come back to where we are I think that will be very non-obvious.